cardpresent/src/main/java/com/fattmerchant/android/Mock/MockDriver.kt

```diff
@@ -0,0 +1,161 @@
+package com.fattmerchant.android.Mock
+
+import com.fattmerchant.android.chipdna.ChipDnaDriver
+import com.fattmerchant.omni.MobileReaderConnectionStatusListener
+import com.fattmerchant.omni.SignatureProviding
+import com.fattmerchant.omni.TransactionUpdateListener
+import com.fattmerchant.omni.UserNotificationListener
+import com.fattmerchant.omni.data.*
```

Comment:
we need to start avoiding wide card import..

Reply:
Addressed in d57929f71e8dff91dd42a4e74d7e4a1198bea3d5